### PR TITLE
[1LP][RFR] Moving SCVMM manual cases

### PR DESCRIPTION
**Pull request by john-dupuy**
Purpose or Intent
=================

Move manual cases from `test_manual` to `test_scvmm_specific`. Also removing test cases which are automated (`test_retirement`) and removing manual cases that are already in integration tests.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 778e900c0fdd9d0d5e3f34a35122bc551e09e3f3:

`cfme/tests/infrastructure/test_scvmm_specific.py`:
- :red_circle: Line 182:2: **F821** *undefined name 'test_requirements'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/infrastructure/test_scvmm_specific.py [hunk 1]
```diff
@@ -27,3 +28,193 @@ def test_no_dvd_ruins_refresh(provider, testing_vm_without_dvd):
...
+def test_check_disk_allocation_size_scvmm():
+    """
+    https://bugzilla.redhat.com/show_bug.cgi?id=1490440
+    Steps to Reproduce:
+    1.Provision VM and check it"s "Total Datastore Used Space"
+    2.go to VMM and create Vm"s Checkpoint
+    3.open VM Details check - "Total Datastore Used Space"
+
+    Polarion:
+        assignee: apagac
```
Comment (jawatts) [inline cfme/tests/infrastructure/test_scvmm_specific.py]: should this be on you?

Comment (john-dupuy) [reply cfme/tests/infrastructure/test_scvmm_specific.py]: yup, missed that one

File: cfme/tests/infrastructure/test_scvmm_specific.py [hunk 2]
```diff
@@ -27,3 +28,193 @@ def test_no_dvd_ruins_refresh(provider, testing_vm_without_dvd):
...
+        caseimportance: medium
+        initialEstimate: 1/8h
+        startsin: 5.7
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_provider_summary_scvmm():
```
Comment (digitronik) [inline cfme/tests/infrastructure/test_scvmm_specific.py]: I would parametrize for providers `scvmm` type

File: cfme/tests/infrastructure/test_scvmm_specific.py [hunk 3]
```diff
@@ -27,3 +28,193 @@ def test_no_dvd_ruins_refresh(provider, testing_vm_without_dvd):
...
+        caseimportance: medium
+        initialEstimate: 1/4h
+        startsin: 5.4
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+def test_host_info_scvmm2016():
```
[the remark this answers is not in the document]
Comment (john-dupuy) [reply cfme/tests/infrastructure/test_scvmm_specific.py]: Will parametrize in a later PR when we can start testing SCVMM.

**Pull request merged**